README.md

```diff
@@ -127,9 +127,13 @@ The following steps are only relevant if you intend to contribute to the reposit
 
 	  tox -e py37
 
-- To run linters:
+- To run linters (code style checks):
 
-	  tox -e flake8
+    tox -e flake8
+
+- To run static type checks:
+
+    mypy sandbox scripts simulation/v1 tac tests templates
```

Comment:
Shall we use `tox -e mypy`, once added the command in the `tox.ini`, like [here](https://github.com/fetchai/agents-aea/pull/61/files#diff-b91f3d5bd63fcd17221b267e851608e8)?

Reply:
@marcofavorito yes